images/openwisp_freeradius/Dockerfile

```diff
@@ -1,8 +1,7 @@
-FROM freeradius/freeradius-server:3.0.26-alpine
+FROM freeradius/freeradius-server:3.2.2-alpine
 
 # hadolint ignore=DL3018
-RUN apk add --no-cache --update tzdata~=2022f-r1 postgresql-dev~=13.8-r0 \
-                                postgresql-client~=13.8-r0 && \
+RUN apk add --no-cache --update tzdata postgresql-dev postgresql-client && \
```

Comment:
why unstick the depencies?

Reply:
Hi Bixilion, I don't have a good answer for that. We are running psql15 so I just wanted as new client as possible.

Reply:
we should not do this, we have run into a lot of trouble with unsticked dependencies: the building process becomes very unstable, it breaks periodically because some new dependency introduces changes which do not play well with the process.
The upgrade of the dependency has to be done periodically.

Reply:
I have no strong opinion on that so I reverted my changes.